libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -326,6 +326,15 @@ bool RC_Channel::in_trim_dz() const
     return is_bounded_int32(radio_in, radio_trim - dead_zone, radio_trim + dead_zone);
 }
 
+
+/*
+   return trues if input is within deadzone of min
+*/
+bool RC_Channel::within_min_dz() const
+{
+    return is_bounded_int32(radio_in, radio_min - dead_zone, radio_min + dead_zone);
```

Comment:
I wonder if this should just be:
```suggestion
    return radio_in < radio_min + dead_zone;
```
Seems a bit odd to apply the deadzone on the low side of min.